<?xml version="1.0" ?>

<launch>

  <arg name="robot_ip" default="130.230.37.115"/>
  <arg name="load_gripper" default="true" />
  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>


  <include file="$(find panda_moveit_config)/launch/move_group.launch">
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>


    <rosparam command="load" file="$(find cobot_controllers)/config/configuration.yaml" />
    <node name="custom_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="joint_position_controller"/>

</launch>
